src/MineOnlineBroadcast.java

```diff
@@ -174,47 +175,39 @@ public void enable() {
+                while (true) {
                     if (System.currentTimeMillis() - MineOnlineBroadcast.lastPing > 45000) {
                         lastPing = System.currentTimeMillis();
-                        try {
-                            Properties propertiesFile = new Properties();
-                            propertiesFile.load(new FileInputStream(new File("server.properties")));
-
-                            boolean isPublic = propertiesFile.getProperty("public", "true").equals("true");
-                            if(!isPublic)
-                                return;
-
-                            String ip = propertiesFile.getProperty("serverlist-ip", propertiesFile.getProperty("server-ip", propertiesFile.getProperty("ip", null)));
-                            String port = propertiesFile.getProperty("serverlist-port", propertiesFile.getProperty("server-port", propertiesFile.getProperty("port", "25565")));
-                            int users = etc.getServer().getPlayerList().size();
-                            int maxUsers = Integer.parseInt(propertiesFile.getProperty("max-players", "20"));
-                            String name = propertiesFile.getProperty("server-name", "Minecraft Server");
-                            boolean onlineMode = propertiesFile.getProperty("online-mode", "true").equals("true");
-                            //String md5 = propertiesFile.getProperty("version-md5", "");
-                            boolean whitelisted = propertiesFile.getProperty("whitelist", "false").equals("true");
-                            boolean dontListPlayers = propertiesFile.getProperty("dont-list-players", "false").equals("true");
-                            String motd = propertiesFile.getProperty("serverlist-motd", null);
-
-                            String[] playerNames = etc.getServer().getPlayerList().stream().map(player -> player.getName()).collect(Collectors.toList()).toArray(new String[users]);
-
-                            listServer(
-                                    ip,
-                                    port,
-                                    users,
-                                    maxUsers,
-                                    name,
-                                    onlineMode,
-                                    md5,
-                                    whitelisted,
-                                    playerNames,
-                                    motd,
-                                    dontListPlayers
-                            );
-                        } catch (IOException ex) {
-                            //ex.printStackTrace();
-                            // ignore.
-                        }
+                        boolean isPublic = mineOnlineConfig.getConfigBoolean("public");
+                        if (!isPublic)
+                            return;
+
+                        String ip = mineOnlineConfig.getConfigString("serverlist-ip");
```

Comment:
This should fallback to the server properties if no ip/port is provided in the config.